src/server/routes/forwardingApp.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2018-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { taxonomyLookup } from '../helpers/taxonomyLookup';
+import { getToken } from '../helpers/auth';
+import { storeAccessToken } from '../../util/apiHelpers';
+
+export async function forwardingApp(req, resp, next) {
+  const token = await getToken();
+  storeAccessToken(token.access_token);
+
+  const pathParts = req.originalUrl.substring(1).split('/');
+  const languagePrefix =
+    pathParts.length > 0 && pathParts[0] !== 'node' ? `/${pathParts[0]}` : '';
+
+  const requestUrl = `ndla.no${req.originalUrl}`;
+
+  await taxonomyLookup(requestUrl)
```

Comment:
Use async await instead of then resolving:
```
try {
   const newPath =  await taxonomyLookup(requestUrl);
   resp.redirect(301, `${languagePrefix}/subjects${newPath.path}`)
} catch () {
   next();
}
```